src/scorer.py

```diff
@@ -125,8 +133,28 @@ def diff_insert(line: str) -> None:
         def diff_delete(line: str) -> None:
             deletions.append(line)
 
-        self.differ.set_seq1([line.mnemonic for line in cand_seq])
-        result_diff = self.differ.get_opcodes()
+        if self.algorithm == "levenshtein":
+            import Levenshtein
+
+            remapping = dict()
+
+            def remap(seq: List[str]) -> str:
```

Comment:
Having slept on it, I think I get why this remapping is necessary.
Given 2 sequences of instruction mnemonics:
```
a=["lw", "sll", "sw"]
b=["lw", "sw", "sw"]
```
I'd be tempted to do:
```
result = Levensthein.op_codes("".join(a), "".join(b))
```
But that'll cause the perceived diff to be larger than one. We want to know the number mnemonics that are different, not the number of characters that differ between the lists of mnemonics.

So this function gives each mnemonic a unique number (in a pretty novel fashion) and converts that number directly into ascii. Now the Levenshtein algorithm will give us back a diff count that matters for our purposes